test/functionalTest/cases/0999_update_single_attr_value_by_eid/update_single_attr_value_by_eid_error_handling.test

```diff
@@ -0,0 +1,100 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Error handling For PUT Individual Entity Attribute
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. PUTT /v2/entities/E1/attrs/A1/value to see BAD METHOD, only GET, PUT 
+# 02. PUT /v2/entities/E1/attrs/A1/value without 'value' in payload
+# 03. PUT /v2/entities/E1/attrs/A1/value with 'value' and 'value2' in payload
```

Comment:
Sure, it's the same though - everything but 'value' gives an error.
However, 'type' is more interesting than 'asdasd' or 'value2'.